### ISSUE TITLE
P30 pro USB troubles after downgrading to EMUI 10

### ISSUE THREAD

**Issue by avikrem** [ProfessorJTJ/HISuite-Proxy#220]
After a overnight update, the phone went into total lockdown and no possible way to get it working again.  Tried Factory Reset, Hisuite didnt recognice it and ERecovery didnt work. 
I found a v10.1.0.150 firmware with correct regioncodes and downgraded to this.   But then i lost the USB connection and screen charging blobbs when charging.  It only charges from my PC.
I have now been able to update it to Emui 11 but the usb connection and charging blobs are still not working. 
I have all different USB options in the Dev screen but still not working. 
i see in the Alert note in the Code selection, that this happens if you downgrade past EMUI 11 but it works again if you update again, but for me it dont. 
Anyone had this problem before?

**Comment by ProfessorJTJ** [comment]
DId you receive the OTA update inside your phone or used a third party app to upgrade?

Are you on latest available firmware or you're just on old versions of EMUI 11?

**Comment by avikrem** [comment]
I did get the updates on OTA inside the phone.    I also had to use the eRecovery on some of the updates to work, as i got lots of troubles on some of the updates.  
At this moment i am actually updating the troublesome phone to  EMUI 12  12.0.0.132(C431E3R2P2)  from EMUI11 11.0.0.174
I got 2 identical phones where one is abit broken, and its the same update being on them both when its finish updating now.
